src/LaunchDarkly/Impl/Integrations/CurlEventPublisher.php

```diff
@@ -80,7 +80,7 @@ private function createCurlArgs(string $payload): string
     {
         $scheme = $this->_ssl ? "https://" : "http://";
         $args = " -X POST";
-        $args.= " --connect-timeout " . $this->_connectTimeout;
+        $args.= " --max-time " . $this->_connectTimeout;
```

Comment:
The `LDClient` config already allows specifying both a `connect_timeout` and a max length `timeout` parameter. We use that `timeout` parameter in the existing Guzzle client.

Can you update this so that we still support the connect time parameter as before, and also add the max time flag hooked up to that value?